test/cucumber/support/world.js

```diff
@@ -193,7 +196,7 @@ class World {
 
   // Debugging helper
   inspect(data) {
-    if (typeof data === 'object') {
+    if (data !== null && typeof data === 'object') {
```

Comment:
Added explicit comparison to `null` to prevent false positives (`typeof null` is `"object"`).